src/inlineBlockList.story.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import InlineBlockList from './InlineBlockList';
+import { storiesOf } from '@kadira/storybook';
+import { InfoWrapper } from './utils/storyComponents';
+
+const basicItems = [
+	'English',
+	'English (Australian)',
+	'Deutsch',
+	'Español',
+	'Español (España)',
+	'Français',
+	'Italiano',
+	'Nederlands',
+	'Português',
+	'日本語',
+	'한국어',
+];
+
+storiesOf('InlineBlockList', module)
+	.addWithInfo(
+		'default',
+		'Basic usage of InlineBlockList',
+		() => (
+		<InfoWrapper>
+			<InlineBlockList
+				style={{padding: '20px'}}
```

Comment:
I'd prefer to import a `Section` & `Chunk` here to an inline style. Showing layout components in the examples (even if they're unrelated to `InlineBlockList`) seems like it would be helpful